tests/interactive/test_get_all_available_passes.py

```diff
@@ -62,4 +62,6 @@ def test_get_all_available_passes():
         rewrite_by_names_dict=individual_rewrite.REWRITE_BY_NAMES,
     )
 
-    assert res == expected_res
+    for e in expected_res:
+        assert e in res
+    assert len(res) < len(all_passes)
```

Comment:
Comparing a dynamic list (available passes) to a fixed list (expected available passes) results in test failure when adding passes. Possible solutions:
1. Make both available and expected available passes fixed lists, e.g. by filtering `all_passes` to only contain passes known at the time of creating the test.
2. Make both available and expected available passes dynamic lists, e.g. by testing that each available pass actually has an effect.
3. Soften the check (shown above).

Reply:
@superlopuh

Reply:
I disagree with your change, and agree with your sentiment, if you have the bandwidth to fix this test then it would be welcome. In the meantime, could you please update the expected pass list instead?

Reply:
I disagree with the method of testing this and cannot offer the change requested.

Reply:
I'll just fix the test today, it's been made much easier recently since Alex fixed the underlying infrastructure, thanks for providing a good reason to do it now rather than later.

Reply:
Thanks, I'm rolling back changes to this file in this PR.